packages/patternfly-4/react-charts/src/components/ChartDonut/ChartDonut.js

```diff
@@ -1,21 +1,96 @@
+import ChartLabel from '../ChartLabel/ChartLabel';
+import { default as ChartDonutTheme, styles } from '../ChartTheme/themes/theme-donut';
 import ChartTooltip from '../ChartTooltip/ChartTooltip';
 
 export const propTypes = {
   /**
    * See TypeScript API docs: https://formidable.com/open-source/victory/docs/victory-area/
    */
-  '': PropTypes.any
+  '': PropTypes.any,
+  /**
+   * The height props specifies the height the svg viewBox of the chart container. This value should be given as a
+   * number of pixels.
+   *
+   * Because Victory renders responsive containers, the width and height props do not determine the width and
+   * height of the chart in number of pixels, but instead define an aspect ratio for the chart. The exact number of
+   * pixels will depend on the size of the container the chart is rendered into.
+   *
+   * Note: innerRadius may need to be set when using this property.
+   */
+  height: PropTypes.number,
+  /**
+   * The subtitle for the donut chart
+   */
+  subTitle: PropTypes.string,
+  /*
+   * Specifies the theme color; blue (default), green, or multi-color. Overridden by the theme property.
+   */
+  themeColor: PropTypes.string,
+  /*
+   * Specifies the theme variant; 'dark' or 'light' (default). Overridden by the theme property.
+   */
+  themeVariant: PropTypes.string,
+  /**
+   * The title for the donut chart
+   */
+  title: PropTypes.string,
+  /**
+   * The width props specifies the width of the svg viewBox of the chart container. This value should be given as a
+   * number of pixels.
+   *
+   * Because Victory renders responsive containers, the width and height props do not determine the width and
+   * height of the chart in number of pixels, but instead define an aspect ratio for the chart. The exact number of
+   * pixels will depend on the size of the container the chart is rendered into.
+   *
+   * Note: innerRadius may need to be set when using this property.
+   */
+  width: PropTypes.number
 };
 
+const ChartDonut = ({
+  height = ChartDonutTheme.pie.height,
+  innerRadius = ((height || width) - 38) / 2,
+  standalone = false,
+  subTitle,
+  theme = getDonutTheme(themeColor, themeVariant),
+  themeColor,
+  themeVariant,
+  title,
+  width = ChartDonutTheme.pie.width,
+  ...rest
+}) => {
+  const chart = (
+    <React.Fragment>
+      <ChartLabel
+        style={[styles.label.title, styles.label.subTitle]}
+        text={[title, subTitle]}
+        textAnchor="middle"
+        verticalAnchor="middle"
+        x={width / 2}
+        y={height / 2}
+      />
+      <VictoryPie
+        height={height}
+        innerRadius={innerRadius > 0 ? innerRadius : ChartDonutTheme.pie.innerRadius}
+        labelComponent={<ChartTooltip theme={theme} />}
+        standalone={false}
+        theme={theme}
+        width={width}
+        {...rest}
+      />
+    </React.Fragment>
+  );
+  return standalone ? chart : <ChartContainer width={width} height={height}>{chart}</ChartContainer>;
```

Comment:
This logic is reversed from the Victory behavior. `standalone === true` means that the component will be wrapped in an svg, `standalone === false` means it will not.

Reply:
Right. Mixed up the standalone default value.